README.rst

```diff
@@ -94,21 +83,20 @@ Alternatively, clone and build from source:
 When building from source, the closed-source component still needs to be
 installed as above.
 
-.. installation-end-marker
-
 
 License
 =======
 
 Released under the Apache License 2.0. See `<LICENSE>`_ file.
 
-Visualization component released under the 
-`D-Wave EULA <https://docs.ocean.dwavesys.com/en/stable/licenses/inspector.html>`_.
+Visualization component released under the
+`D-Wave EULA <https://docs.dwavequantum.com/en/latest/licenses.html>`_.
+
 
 Contributing
 ============
 
-Ocean's `contributing guide <https://docs.ocean.dwavesys.com/en/stable/contributing.html>`_
+Ocean's `contributing guide <https://docs.dwavequantum.com/en/latest/ocean/ocean_contribute.html>`_
```

Comment:
why do we need `/ocean/ocean_contribute.html`?

Reply:
mea culpa
The file naming could have taken folder into account but I missed that

Reply:
Updated [here](https://github.com/dwavesystems/dwave-ocean-sdk/pull/324/commits/f5ab5b6569996d8ab997dc0778ceb45081cc59bf)